Plutarch/Api/V1/AssocMap.hs

```diff
@@ -407,7 +409,7 @@ instance
   where
   inv a = pmap # plam PlutusTx.inv # a
 
-{- | Combine two 'PMap's applying the given function to any two values that
+{- | Union two 'PMap's applying the given function to any two values that
```

Comment:
Union doesn't seem to be a verb, so I'd word those as "build the union/intersection of".